vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -33,6 +33,7 @@ import hudson.AbortException
          * @parentConfigKey jenkinsKubernetes
          */
         'inheritFrom',
+        'resources',
```

Comment:
This parameter should be part of the container configuration. Probably we may need to add it somewhere after [this](https://github.com/SAP/jenkins-library/pull/1993/files#diff-fcf3f6611af29e44d7787f9bd3cc0345R52).

Reply:
> This parameter should be part


@srinikitha09 You mean the resources should be part of the `STEP_CONFIG_KEYS`, like e.g. `containerPullImageFlags`, `containerShell` which are `maps<containerName, $whatever>` in the same way like the resources map?

But this we have already. You can specify it either via `config.general.jenkinsKubernetes` or in the same way like e.g. `containerPullImageFlags`. See [here](https://github.com/SAP/jenkins-library/pull/1993/files#diff-fcf3f6611af29e44d7787f9bd3cc0345R172).

Additionally it can be configured in `general.jenkinsKubernetes`. In case you think that disturbs I can remove it.